.env.test

```
@@ -1,4 +1,6 @@
+AUTOFORM_URL=https://autoform.ekosystem.slovensko.digital
 AUTOFORM_JAVASCRIPT_URL=https://autoform.ekosystem.slovensko.digital/assets/autoform.js
 AUTOFORM_ACCESS_TOKEN=NOTNEEDEDFORTESTS
+AUTOFORM_PRIVATE_ACCESS_TOKEN=NOTNEEDEDFORTESTS
```

nedavajme tu taketo vselico co neviem ani precitat

```suggestion
AUTOFORM_PRIVATE_ACCESS_TOKEN=
```